docs/aws/services/dynamodb-pricing.md

```diff
@@ -0,0 +1,121 @@
+| Write Type     | `Standard, Transactional`                                   | Transactional operations allow *ACID guarantees* at twice the standard cost                                                                                 | [Docs](https://aws.amazon.com/blogs/aws/new-amazon-dynamodb-transactions/)                                                         |
+| Read Type      | `Eventually Consistent, Strongly Consistent, Transactional` | Dynamo reads are by default *Eventually Consistent* - when you read from a table, the response might not reflect the results of a recently completed write. | [Docs](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadConsistency.html)                           |
+| Read Operation | `GetItem, Scan, Query`                                      | Scans return the entire contents of a table; Queries allow a much faster & cheaper read of a subsection of the table                                        | [Docs](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/bp-query-scan.html)                                        |
+| Indexes        | `None, Local Secondary Index, Global Secondary Index`       | Indexes allow an alternate set of `partition_key` + `sort_key` to be used for queries                                                                       | [Overview](https://www.dynamodbguide.com/secondary-indexes/)                                                                       |
+
+
+## Billing Mode
+
+"Use on-demand until it hurts" - [Alex DeBrie, quoting Jared Short](https://twitter.com/geoff_baskwill/status/1421181922097737729)
+
+Provisioned Throughput is cheaper if you have a meaningful number of reads/writes *distributed evenly across time*. Any reads/writes above the provisioned threshold *will fail*, so it is not well suited to bursty or unpredictable workloads.
+
+Provisioned Throughput includes optional Auto-Scaling if throughput thresholds are being exceeded ([Docs](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/AutoScaling.html)).
+
+[AWS Free Tier](https://aws.amazon.com/free) includes 25 reads/writes-per-second of Provisioned Throughput (across any # of tables), but *does not* include any On Demand mode usage.
+
+
+## Write Type
+
+Standard writes are relatively straightforward and include single item writes (`table.put_item`) and batch writes (`batch.put_item`).
+
+Transactions (`client.execute_transaction`) group up to 25 writes (or reads, updates, or deletes) together and *guarantee that they succeed or fail together.*
+
+For a given write of an item up to *1 KB* in size:
+
+| Type                 | Cost                       |
+|----------------------|----------------------------|
+| Standard single item | 1 WRU                      |
+| Standard batch       | 1 WRU (per item)           |
+| Transactional        | 2 WRU (2x)                 |
+| Oversize 4 KB item   | 4 WRU (4x, size dependent) |
```

Comment:
Is this supposed to be `WCU`? I actually can't find a reference to `WRU` anywhere on the pricing page but that doesn't mean I'm right. 😄 

Also for acronyms like this, the first time they're referenced we try to write out the full name and then put the acronym in parenthesis to educate folks as to what it stands for.

Reply:
The acronym WRU appears to mostly be used by third party vendors and how-to guides. I added a section under BillingMode to clarify the relevant units for each Billing mode type
![image](https://user-images.githubusercontent.com/23533756/127756621-9a82efe2-18b3-451b-8be6-be83e96a4c71.png)